frontend/javascripts/oxalis/view/right-menu/tree_hierarchy_view.js

```diff
@@ -353,6 +365,58 @@ class TreeHierarchyView extends React.PureComponent<Props, State> {
       const rgbColorString = tree.color.map(c => Math.round(c * 255)).join(",");
       // Defining background color of current node
       const styleClass = this.getNodeStyleClassForBackground(node.id);
+      const menu = (
+        <Menu>
+          <Menu.Item key="changeTreeColor">
+            <div style={{ position: "relative", display: "inline-block", width: "100%" }}>
+              <i
+                className="fas fa-eye-dropper fa-sm"
+                style={{
+                  cursor: "pointer",
+                  color: Utils.rgbToHex(Utils.map3(value => value * 255, tree.color)),
+                }}
+              />{" "}
+              Change the color
```

Comment:
In accordance with the next menu entry "Shuffle Tree Color", I would propose to name this
```suggestion
              Select Tree Color
```